src/Action/Woopit/CreateDeliveryTrait.php

```diff
@@ -23,6 +23,22 @@ protected function createDelivery(WoopitQuoteRequest $data): Delivery
         $pickup = $this->createTask($data->picking, Task::TYPE_PICKUP);
         $dropoff = $this->createTask($data->delivery, Task::TYPE_DROPOFF);
 
+        if ($data->packages) {
+            $packagesString = '';
+
+            foreach($data->packages as $package) {
+                if (!empty($packagesString)) {
+                    $packagesString .= ', ';
+                }
+                $packagesString .= $package['quantity'];
+                if (isset($package['weight'])) {
+                    $packagesString .= ' x ' . $package['weight']['value'] . ' ' . $package['weight']['unit'];
+                }
+            }
+
+            $pickup->setComments(sprintf('Packages: %s', $packagesString));
+        }
+
```

Comment:
The same code seems to be repeated both in `CreateDeliveryTrait` and `UpdateDeliveryTrait`. Can we factorize it?